support/packages/local-npm/lib/index.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+/* eslint-disable @liferay/no-dynamic-require, no-console */
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const LOCAL_REGISTRY = 'http://localhost:4873';
```

Comment:
Maybe add a comment on why this port is choesen. Pointing back to verdaccio's docs would probably be enough

Reply:
This really depends on how we install Verdaccio, so it should probably be configurable. But I wanted to see first how people decides/tries to install Verdaccio, then decide what to do with this....